modules/scripts/startup-script/files/running-script-warning.sh

```diff
@@ -13,17 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-WARNING=$(
-	cat <<EOF
-** NOTICE **: System services may not be running until startup scripts complete.
-The output of the command below will end with "Finished Google Compute Engine 
-Startup Scripts." when they are complete. Please review the output for any 
-errors which may indicate that services are unhealthy. 
+SCRIPT_COMPLETE_FILE="/run/startup_script_msg"
 
-sudo journalctl -b 0 -u google-startup-scripts.service
-EOF
+WARNING_MESSAGE=$(
+	cat <<-EOF
+		** NOTICE **: HPC-Toolkit startup scripts should start soon.
+		Systems may not be configured or running.
+	EOF
 )
 
-echo
-echo "${WARNING}"
-echo
+if [ -f "${SCRIPT_COMPLETE_FILE}" ]; then
+	echo
+	cat "${SCRIPT_COMPLETE_FILE}"
+	echo
+else
+	echo
+	echo "$WARNING_MESSAGE"
```

Comment:
Testing with this blueprint, I still observe the "should start soon" message at login well after successful completion of the script. I saw the wall message indicating successful completion and then logged in again. I suspect this is the problem.

I thought intent was for this file to be long-lived and always indicate the status of the previous run.

```yaml
---

blueprint_name: pr1956

vars:
  deployment_name: ss
  region: us-central1
  zone: us-central1-c

deployment_groups:
- group: first
  modules:
  - id: network
    source: modules/network/pre-existing-vpc
  - id: script
    source: modules/scripts/startup-script
    settings:
      runners:
      - type: shell
        destination: sleep.sh
        content: |
          #!/bin/bash
          sleep 100
  - id: repro
    source: modules/compute/vm-instance
    use:
    - network
    - script
    settings:
      machine_type: c2-standard-4
```

Reply:
My mistake, this didn't work because I hadn't linked vm-instance and two other modules in the `modules` folder.  Fixed.